src/lifx/light.js

```diff
@@ -71,9 +71,9 @@ Light.prototype.color = function(hue, saturation, brightness, kelvin, duration,
   validate.optionalCallback(callback, 'light color method');
 
   // Convert HSB values to packet format
-  hue = Math.round(hue / constants.HSBK_MAXIMUM_HUE * 65535);
-  saturation = Math.round(saturation / constants.HSBK_MAXIMUM_SATURATION * 65535);
-  brightness = Math.round(brightness / constants.HSBK_MAXIMUM_BRIGHTNESS * 65535);
+  hue = Math.round((hue / constants.HSBK_MAXIMUM_HUE) * 65535);
+  saturation = Math.round((saturation / constants.HSBK_MAXIMUM_SATURATION) * 65535);
+  brightness = Math.round((brightness / constants.HSBK_MAXIMUM_BRIGHTNESS) * 65535);
```

Comment:
Revert these unrelated changes. It'll be really confusing to see `git blame` say you've last edited the lines but the commit says something about "TileApi".

This would improve the readability a bit I agree, but as this `* 655535` part is repeating so much it would make more sense to refactor it to be done in a function anyway.

Reply:
I revert this changes, but it drives me crazy.... infact nobody normal knows the order of execution of the operators * /.